x/emissions/module/rewards/rewards.go

```diff
@@ -90,7 +90,11 @@ func EmitRewards(ctx sdk.Context, k keeper.Keeper, blockHeight BlockHeight, weig
 			continue
 		}
 	}
-	if !totalReward.IsZero() {
+	blocksPerMonth, err := k.GetParamsBlocksPerMonth(ctx)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get blocks per month")
+	}
+	if !totalReward.IsZero() && uint64(blockHeight)%blocksPerMonth == 0 {
```

Comment:
Rewards should be continuously doled out though, right? so we do need to write this data somewhere each block, no?

Reply:
This if statement singularly controls the writing of (reputer component of) the f_stakers value in the mint module. The if statement has no effect on the rewards being doled out or not

f_stakers is used monthly so there's no need to update it every block when we're only using it once per month. We could do some complicated sum or average calculation over the course of the month that seems like too complicated for what is an indirect value